### PR TITLE
Changed README for better readability.

### PR DESCRIPTION
**Pull request by v0lkan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,37 +118,36 @@ Commands:
 
 Options:
 
-  -v, --version     显示版本信息
```
Comment (v0lkan) [inline README.md]: removed the chinese characters; the default help should better be in English for a global audience.

File: README.md [hunk 2]
```diff
@@ -200,3 +206,9 @@ Thanks to the authors of the above libraries to provide such a useful library.
 
 This project is licensed under the MIT License - see the [LICENSE](https://github.com/hiproxy/hiproxy/blob/master/LICENSE) file for details
 
+## Code of Conduct
```
Comment (v0lkan) [inline README.md]: Added link to code of conduct.

**Comment by zdying** [comment]
@v0lkan Thank you very much.

**Comment by v0lkan** [comment]
@zdying happy to help :)

**Pull request merged**